### PR TITLE
Catching down testJournal.testBasic flake

### PR DESCRIPTION
**Pull request by marusak**
There is race that when stop() is called, it stops existing processes but
right after it a new process is created and thus it keeps running forever.
Normally this would not be a problem, but with this weird
combination of JS + React it can happen that messages from this service
are shown in the UI.

Honestly, the best solution would be to write it fully in React, I am really playing with the idea of doing that.

### CODE REVIEW

**Comment by marusak** [comment]
On all OSes only once testJournal.testBasic was retried:
https://logs.cockpit-project.org/logs/pull-15155-20210115-140451-fc6cabaf-ubuntu-2004/log.html#307-1
But this is a completely new failure, but I might have an idea, so I'll take a look and push here.

**Comment by marusak** [comment]
Ok, not a single `check-journal` failure nor retry in any OS, force triggering all tests once again.

File: test/verify/check-journal [hunk 1]
```diff
@@ -136,7 +136,7 @@ ExecStart=/bin/sh -c '/usr/bin/seq 123; sleep 10'
 Description=Slowly log 10 identical lines
 
 [Service]
-ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 10'
+ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 10'
```
Comment (martinpitt) [inline test/verify/check-journal:139]: Ugh, that feels like a systemd journal bug. I don't really like sleeps, but it's harmless enough for now. Thanks!

**Review by martinpitt — APPROVED** [review]
Thanks for pinning this down!

**Comment by marusak** [comment]
3 runs, in each all OSes passed check-journal 3 times. Seems to be working, but let's try it out once more :crossed_fingers:

File: test/verify/check-journal (hunk 1 above)
Comment (marusak) [reply test/verify/check-journal:139]: It does a bit but it is super tight race, so almost not possible to create any reasonable reproducer except: run in loop until it hopefully fails.

**Comment by marusak** [comment]
Green in 4 tries, each OS ran check-journal 12 times.

**Pull request merged**